### DISCUSSION TITLE
Forgotten fish not on map.

### DISCUSSION THREAD

**Discussion by Wild-Durango** [janstehno/cotw-castmate#8] [Issues]
Ello Devs of the COTW ANGLER COMPANION APP I noticed on the Golden Reserve you guys forgot to include the Signal Crayfish to the list and map, not sure if that really was missed or decided not to be added I'm not sure, just wanted to bring this up and see if it could be added. Thanks

**Comment by janstehno** [comment]
Hey, I am aware of the missing fish on the maps for Golden Ridge Reserve. I plan to update the locations when I release the Japan update (according to the #6 issue) in late January.

**Reply by ram64** [reply]
Hi there and Merry Christmas. Do you get the locations from the game files or manually adding them? I can help if the latter.

**Reply by janstehno** [reply]
I am copying fish locations from official Discord and their Resource Hub [here](https://docs.google.com/spreadsheets/u/0/d/1grFzdd-oAt2JKgKAqdMLQPwGmdGwPqDt624OKi-9n70/htmlview#gid=65439218). There are also missions and collectibles, but I do not know if there are any relevant ones on the new reserve. But if there are, it would be nice to have them. I will do the missions and try to find all collectibles anyway, so it is up to you. Anything helps truly.
